### PR TITLE
Login navigation bar disappears during a pop transition

### PR DESCRIPTION
**Pull request by salimbraksa**
## Related PR
This PR is blocking the following PR:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/19461

## Description
Please check the related the PR for more details on the issue and the test instructions.

### CODE REVIEW

**Review by twstokes — APPROVED** [review]
LGTM @salimbraksa 🚀 

I followed the testing steps in https://github.com/wordpress-mobile/WordPress-iOS/pull/19461 and all passed.

File: WordPressAuthenticator.podspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressAuthenticator'
-  s.version       = '3.2.1'
+  s.version       = '3.2.2'
```
Comment (Gio2018) [inline WordPressAuthenticator.podspec]: The versioning here should follow our guidelines and issue a new beta. I'll share more details on slack, thank you!

**Review by Gio2018 — CHANGES_REQUESTED** [review]
Hey @salimbraksa , we should stick to our versioning guidelines. Let's discuss this in more details on Slack!

File: WordPressAuthenticator.podspec (hunk 1 above)
Comment (salimbraksa) [reply WordPressAuthenticator.podspec]: @Gio2018 Updated the podspec version to `3.2.2-beta.1` 9bee712d8117e28432c4c1e1eb1fbb89d6156b12

**Review by Gio2018 — APPROVED** [review]
:shipit:

**Pull request merged**